ixmp4/core/region.py

```diff
@@ -87,48 +91,40 @@ def create(
         name: str,
         hierarchy: str,
     ) -> Region:
-        model = self.backend.regions.create(name, hierarchy)
-        return Region(_backend=self.backend, _model=model)
+        dto = self._backend.regions.create(name, hierarchy)
+        return Region(backend=self._backend, dto=dto)
 
     def get(self, name: str) -> Region:
-        model = self.backend.regions.get(name)
-        return Region(_backend=self.backend, _model=model)
+        dto = self._backend.regions.get_by_name(name)
+        return Region(backend=self._backend, dto=dto)
 
     def delete(self, x: Region | int | str) -> None:
         if isinstance(x, Region):
             id = x.id
         elif isinstance(x, int):
             id = x
         elif isinstance(x, str):
-            model = self.backend.regions.get(x)
-            id = model.id
+            dto = self._backend.regions.get_by_name(x)
```

Comment:
```suggestion
            id = self._backend.regions.get_by_name(x).id
```
Seems clear enough to me and saves a line :)